src/commands/convert/trivy2hdf.ts

```diff
@@ -25,7 +27,23 @@ export default class Trivy2HDF extends Command {
     }
 
     const converter = new Mapper(input)
-    fs.writeFileSync(checkSuffix(flags.output), JSON.stringify(converter.toHdf()))
+    const results = converter.toHdf()
+
+    if (Object.keys(results).length > 1) {
+      _.forOwn(results, (result, filename) => {
+        fs.writeFileSync(
+          path.join(flags.output, checkSuffix(filename)),
+          JSON.stringify(result),
+        )
+      })
+    } else {
+      _.forOwn(results, result => {
```

Comment:
if i'm reading this right, this is potentially returning a single file named after the output path instead of putting that file into a folder. imo we should just consistently shove things into a folder regardless of the number of results. if you wanna keep the single file vs folder paradigm, then the documentation needs to be updated to reflect that.